### DISCUSSION TITLE
OTP Issues

### DISCUSSION THREAD

**Discussion by serv-ify** [supabase/supabase#21808] [Questions]
Hi all,

I would like to have some help if possible; I've integrated my application with Supabase and encountered an issue during OTP verification. It displays a "Login error" message, specifically citing an error in sending the confirmation SMS. I suspect there might be a problem related to the database setup. Could you provide any insights or suggestions?

**Comment by GaryAustin1** [comment]
Not much info provided by you. Not sure how anyone can help.
What is the exact error?
Have you looked in the dashboard logs (maybe auth log) for more detailed errors.
You don't say what provider you are using.

Here are the docs:  https://supabase.com/docs/guides/auth/phone-login
